OracleWebLogic/samples/12213-patch/README.md

```diff
@@ -2,6 +2,8 @@ Example of Image with WLS Domain
 ================================
 This Dockerfile extends the Oracle WebLogic image by applying a patch.
 
+**NOTE**: Oracle strongly recommends using the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-image/) (WIT) with the `--recommendedPatches` option, which automatically downloads and applies all the required PSUs, bundle patches, and one-off patches to your 12.2.1.3, 12.2.1.4, and 14.1.1 images. Note that the 12.2.1.3 images in OCR already have these patches applied.

```

Comment:
automatically downloads and applies all the required PSUs, bundle patches, and one-off patches to your 12.2.1.3, 12.2.1.4, and 14.1.1 images and updates OPatch. Note that the WebLogic CPU images in OCR already have these patches applied.